app/data/helpers/courses.js

```diff
@@ -0,0 +1,89 @@
+const Utils = require('./utils')
```

Comment:
if Utils isn't a constructor then I wouldn't capitalise. Nitpick alert.

Reply:
Changed to lower case